packages/horizon/contracts/staking/HorizonStaking.sol

```diff
@@ -278,6 +278,8 @@ contract HorizonStaking is HorizonStakingBase, IHorizonStakingMain {
         uint256 tokens
     ) external override notPaused {
```

Comment:
Can we add the following to the fn documentation `horizon/contracts/interfaces/internal/IHorizonStakingMain.sol`:

```
     * Requirements:
     * - Caller must have previously approved this contract to pull tokens from their balance.
```